README.md

```diff
@@ -2,8 +2,13 @@
 
 A repository for team interaction, syncing, and handling meeting notes across the JupyterLab ecosystem.
 
-## Weekly Developer Meeting
-Weekly Developer Meetings take place on [zoom](https://zoom.us/my/jovyan?pwd=c0JZTHlNdS9Sek9vdzR3aTJ4SzFTQT09), on Wednesdays at 9AM Pacific Time ([your time](https://www.thetimezoneconverter.com/?t=9%3A00%20am&tz=San%20Francisco)). Minutes will be taken at [Hackmd.io](https://hackmd.io/Y7fBMQPSQ1C08SDGI-fwtg). Each week an issue will be opened with that week's meeting minutes and the issue from the previous week will be closed. 
+## Weekly developer meeting
```

Comment:
Maybe we can make the heading more explicit, for example:

```suggestion
## Weekly JupyterLab developer meeting
```

Reply:
We should even replace _developer_ by _team_ to align with the notes issue (I think this one done on purpose to indicate any one is welcomed to join).